### PR TITLE
Add toolchain file

### PR DESCRIPTION
**Pull request by danforbes**
Add a toolchain file to avoid bugs related to instability in nightly builds.

Closes #76 
Relates to #90 
Closes #91

### CODE REVIEW

**Comment by danforbes** [comment]
This was the result of running `cargo check` with the new toolchain file? Is there some other set up we need to specify in order to get this working?
```
      Finished dev [unoptimized + debuginfo] target(s) in 1m 28s
       Running `/home/parity/Code/node-template/target/debug/wbuild-runner/node-template-runtime13439624246709566360/target/x86_64-unknown-linux-gnu/debug/wasm-build-runner-impl`
  Rust WASM toolchain not installed, please install it!
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by adoerr** [comment]
Are you sure, you have installed the WASM toolchain for the specific nightly version you are using in the toolchain file. I'm using a toolchain file and it works just fine.

**Comment by danforbes** [comment]
This should be rounded out with https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/712

**Comment by wheresaddie** [comment]
I had no issues as long as I assigned the WASM to the nightly match @danforbes please advise if you are still having issues

**Comment by danforbes** [comment]
Yep, that's definitely the missing step...hopefully I have it documented the best way, but I'm more than happy to accept suggestions.

**Review by wheresaddie — APPROVED** [review]
you could add a code comment to verify match with WASM to nightly rust if there are any issues or QA you see popping up-- otherwise looks good. thanks D.

File: README.md [hunk 1]
```diff
@@ -6,18 +6,9 @@ A new FRAME-based Substrate node, ready for hacking :rocket:
...
-
-```bash
-curl https://getsubstrate.io -sSf | bash -s -- --fast
-```
-
-### Manual Setup
-
-Find manual setup instructions at the
+Setup instructions can be found at the
 [Substrate Developer Hub](https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-installation).
```
Comment (bkchr) [inline README.md:12]: There is no `manual-installation` sub-heading.

Comment (danforbes) [reply README.md]: Removed the subheading :+1:

**Pull request merged**